ml_perf/repeat_run.sh

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+set -e
+
+: ${1?"Need to call with 'repeat_run.sh BUCKET_NAME' (no gs:// prefix needed)"}
+
+BUCKET_NAME=$1
+
+while true
+do
+  BASE_DIR=$(pwd)/results/$(hostname)-$(date +%Y-%m-%d-%H)
```

Comment:
It's not actually guaranteed that BASE_DIR will be unique: if someone sets the number of iterations to 1 for some reason, we'll finish multiple loops withing a single hour.

I'm not sure how much we actually care about that, but consider including a unique id to the BASE_DIR.

Reply:
added minutes.